### PR TITLE
replace with new logo

### PR DESCRIPTION
**Pull request by monmonk**

### CODE REVIEW

**Review by jiangzhengwenjz — APPROVED** [review]
Looks good. We're going to leave this until we can test it on real hardware

File: .idea/vcs.xml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (jiangzhengwenjz) [inline .idea/vcs.xml:1]: We definitely don't need git to track this file.

**Review by jiangzhengwenjz — COMMENTED** [review]
I will fix the issue for you this time, but make sure that you only commit files supposed to be tracked when your .gitignore is behind master.

File: .idea/vcs.xml (hunk 1 above)
Comment (monmonk) [reply .idea/vcs.xml:1]: I'm not entirely sure about your first comment. Can you teach me more about it when we meet? thanks a lot for fixing it btw! ^_^

**Pull request merged**